source/math/FloatBox2D.ooc

```diff
@@ -57,14 +57,14 @@ FloatBox2D: cover {
 	}
 	pad: func ~fromFloat (pad: Float) -> This { this pad(pad, pad, pad, pad) }
 	pad: func ~fromSize (pad: FloatSize2D) -> This { this pad(pad width, pad width, pad height, pad height) }
-	pad: func ~fraction (pad: Float) -> This {
-		this pad(pad * this size / 2.0f)
+	enlargeEvenly: func (fraction: Float) -> This {
+		this pad(fraction * (this size width + this size height) / 2.0f)
 	}
-	padFractionAverage: func (pad: Float) -> This {
-		this pad(pad * (this size width + this size height) / 2.0f)
+	enlarge: func (fraction: Float) -> This {
+		this pad(fraction * this size / 2.0f)
```

Comment:
This looks confusing, but I switched places because this one was so alike `shrink`.